.github/workflows/test.yml

```diff
@@ -66,17 +67,40 @@ jobs:
           debug: true
 
   coveralls-finish:
-    needs: build
-    # If pushed / is a pull request against main repo
-    if:
-      (github.event_name == 'push' && github.repository == 'psf/black') ||
-      github.event.pull_request.base.repo.full_name == 'psf/black'
+    needs: main
+    if: github.repository == 'psf/black'
 
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Coveralls finished
+      - name: Send finished signal to Coveralls
         uses: AndreMiras/coveralls-python-action@v20201129
         with:
           parallel-finished: true
           debug: true
+
+  uvloop:
```

Comment:
So does this run in parallel out of the one .yml file? TIL. Nice.

Reply:
Yup, runs in parallel with the main test jobs just like how it did before when it was a separate workflow